tf2/include/tf2/transform_storage.h

```diff
@@ -52,17 +46,18 @@ class TransformStorage
   TF2_PUBLIC
   TransformStorage();
   TF2_PUBLIC
-  TransformStorage(const TimePoint& stamp, const Quaternion& q, const Vector3& t, CompactFrameID frame_id,
-                   CompactFrameID child_frame_id);
+  TransformStorage(
+    const TimePoint & stamp, const Quaternion & q, const Vector3 & t, CompactFrameID frame_id,
+    CompactFrameID child_frame_id);
 
   TF2_PUBLIC
-  TransformStorage(const TransformStorage& rhs)
+  TransformStorage(const TransformStorage & rhs)
   {
     *this = rhs;
   }
 
   TF2_PUBLIC
-  TransformStorage& operator=(const TransformStorage& rhs)
+  TransformStorage & operator=(const TransformStorage & rhs)
   {
 #if 01
```

Comment:
This looks like a bug.  If you don't want to deal with it in this PR, please open an issue to follow-up on it.